src/NKSolver/NKSolvers.F90

```diff
@@ -77,9 +77,8 @@ module NKSolver
 
     subroutine setupNKsolver
 
```

Comment:
I just checked how the options work with the NK. I will leave @anilyil  to review the NK part.